src/index.php

```diff
@@ -5,15 +5,22 @@
 require_once '../vendor/autoload.php';
 require_once 'PlentyConfig.php';
 
+use Findologic\Plentymarkets\Client;
 use Findologic\Plentymarkets\Debugger;
-use Log4Php\Logger;
-use Log4Php\Configurators\LoggerConfigurationAdapterXML;
-
-$configurationAdapter = new LoggerConfigurationAdapterXML();
-Logger::configure($configurationAdapter->convert('Logger/import.xml'));
-
-$log = Logger::getLogger('import.php');
-$customerLogger = Logger::getLogger('import.php');
+use Findologic\Plentymarkets\Exporter;
+use Findologic\Plentymarkets\Registry;
+use Findologic\Plentymarkets\Wrapper\Csv;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+
+function configureLogger(Logger $logger): Logger
+{
+    $logger->pushHandler(new StreamHandler('logs/import.log', Logger::DEBUG));
```

Comment:
:question: The log format is not exactly the same as it was defined in `Logger/import.xml` (and it is in general not possible to exactly replicate log4php messages with Monolog). Is this fine?

Reply:
At least I'm not aware of anyone parsing Plenty-specific log lines.